chore: bump gdk

jackstar12 opened the pull request.

michael1011 reviewed (commented):
Did you test this? Last time we updated GDK, the wallet broke

jackstar12 commented:
> Did you test this? Last time we updated GDK, the wallet broke

Last update broke because of the incorrect max inputs that were imposed, which are now fixed:
https://github.com/Blockstream/gdk/commit/2e53b69eb443ea84c889ac5d86306947a8179b72

We can also remove the fee floor in our implementation since its implemented in gdk properly now:
https://github.com/Blockstream/gdk/commit/1fe1f3b6760b9e9896e0d8ccf8524149f21d9e70

https://github.com/Blockstream/gdk/blob/196340427e061fa9b8f8012672d566b70b8c393c/CHANGELOG.md?plain=1#L79-L80

jackstar12 commented:
Just did a local run of the auto consolidation test with 256 inputs
